SampleUsages/TestScenario.cs

```diff
@@ -34,56 +37,51 @@ internal void RunScenario(ILogger logger, ServerlessResultManager.TestScenario d
             var scenarioType = Tuple.Create(this.Platform, this.TriggerType);
             FunctionTest test = null;
             TriggerTestLoadProfile profile;
-            var inputCount = 0;
+
+            string[] inputItems;
+            try
+            {
+                inputItems = this.Input ?? Directory.GetFiles(this.InputPath);
+            }
+            catch
```

Comment:
Don't use exceptions for normal control flow. Check to see if the input path is a directory or a file and apply the appropriate logic.